tasks/deploy.yml

```diff
@@ -14,7 +14,9 @@
     version: "{{ drupal_deploy_version }}"
     update: "{{ drupal_deploy_update }}"
     force: true
+    depth: "{{ drupal_deploy_depth }}"
```

Comment:
Along with the other comment, this could then be `{{ drupal_deploy_depth | default(omit, true) }}`